Upstream-Status: Pending

Signed-off-by: Maksim Sisov <msisov@igalia.com>
---
From 1da727488fc60b663f9afdc2ad033d5f0edc05fe Mon Sep 17 00:00:00 2001
From: Maksim Sisov <msisov@igalia.com>
Date: Mon, 26 Oct 2020 11:22:09 +0200
Subject: [PATCH] add missing cstring include

fixes:

In file included from ../../third_party/nearby/src/cpp/platform_v2/base/base_input_stream.h:18:
../../third_party/nearby/src/cpp/platform_v2/base/byte_array.h:72:5: error: use of undeclared identifier 'memcpy'; did you mean 'wmemcpy'?
    memcpy(data() + offset, from.data() + source_offset,
    ^~~~~~
    wmemcpy
---
 cpp/platform_v2/base/byte_array.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/third_party/nearby/src/cpp/platform_v2/base/byte_array.h b/cpp/platform_v2/base/byte_array.h
index 6020f2f..2d16153 100644
--- a/third_party/nearby/src/cpp/platform_v2/base/byte_array.h
+++ b/third_party/nearby/src/cpp/platform_v2/base/byte_array.h
@@ -17,6 +17,7 @@
 
 #include <array>
 #include <cstdint>
+#include <cstring>
 #include <string>
 #include <type_traits>
 #include <utility>
-- 
2.25.1

